modules/images-create-s2i-scripts.adoc

```diff
@@ -1,74 +1,79 @@
 // Module included in the following assemblies:
 //
 //* builds/build-strategies.adoc
+// * openshift_images/create-images.adoc
 
 [id="images-create-s2i-scripts_{context}"]
-= Writing S2I scripts
+= Writing source-to-image scripts
 
-You can write S2I scripts in any programming language, as long as the scripts are executable inside the builder image. S2I supports multiple options providing `assemble`/`run`/`save-artifacts` scripts. All of these locations are checked on each build in the following order:
+You can write source-to-image (S2I) scripts in any programming language, as long as the scripts are able to run inside the builder image. S2I supports multiple options providing `assemble`, `run`, and`save-artifacts` scripts. All of these locations are checked on each build in the following order:
 
-. A script specified in the BuildConfig
-. A script found in the application source `.s2i/bin` directory
-. A script found at the default image URL (`io.openshift.s2i.scripts-url` label)
+. A script specified in the BuildConfig.
+. A script found in the application source `.s2i/bin` directory.
```

Comment:
should be  s2i/bin directory, no dot in front of s2i.

Reply:
I saw it's s2i/bin in repo: https://github.com/sclorg/s2i-python-container/tree/master/2.7/